examples/acc-packed.rs

```diff
@@ -0,0 +1,107 @@
+use binius_circuits::{builder::ConstraintSystemBuilder, unconstrained::unconstrained};
+use binius_core::constraint_system::validate::validate_witness;
+use binius_field::{
+	arch::OptimalUnderlier, BinaryField128b, BinaryField16b, BinaryField1b, BinaryField32b,
+	BinaryField8b, TowerField,
+};
+
+type U = OptimalUnderlier;
+type F128 = BinaryField128b;
+type F32 = BinaryField32b;
+type F16 = BinaryField16b;
+type F8 = BinaryField8b;
+type F1 = BinaryField1b;
+
+// FIXME: Following gadgets are unconstrained. Only for demonstrative purpose, don't use in production
```

Comment:
Can you add this kind of warning about `bytes_in` from the linear combination example, please?